packages/xo-server/src/xo-mixins/proxies.js

```diff
@@ -0,0 +1,84 @@
+import synchronized from 'decorator-synchronized'
+import { noSuchObject } from 'xo-common/api-errors'
+
+import Collection from '../collection/redis'
+import patch from '../patch'
+
+const sync = synchronized.withKey((arg1, arg2) => {
+  if (arg1.address !== undefined) {
+    return arg1.address
+  }
+
+  if (arg2.address !== undefined) {
+    return arg2.address
+  }
+
+  return arg1
```

Comment:
@julien-f It have a way to cancel the synchronization in this case?

Reply:
No, `decorator-synchronized` does not provide a way to disable the synchronization for specific keys.